app/src/main/java/com/sun/music61/data/source/TracksRepository.java

```diff
@@ -0,0 +1,41 @@
+package com.sun.music61.data.source;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+public class TracksRepository implements TracksDataSource {
+
+    @Nullable
+    private static TracksRepository sInstance = null;
```

Comment:
default là null rồi e
```suggestion
    private static TracksRepository sInstance;
```